tests/update_workflow_test.go

```diff
@@ -2293,5 +2296,7 @@ func (s *integrationSuite) TestUpdateWorkflow_TimeoutSpeculativeWorkflowTask() {
   9 WorkflowTaskScheduled
  10 WorkflowTaskStarted
  11 WorkflowTaskCompleted
- 12 WorkflowExecutionCompleted`, events)
+ 12 WorkflowExecutionUpdateAccepted
+ 13 WorkflowExecutionUpdateCompleted
+ 14 WorkflowExecutionCompleted`, events)
```

Comment:
Old expectation was based on a bug - we shouldn't be able to complete executions with incomplete updates so either we see UpdateAccepted+UpdateCompleted before the WorkflowExecutionCompleted or the history is invalid.